FiMAdminApi.Models/Models/TruckRoute.cs

```diff
@@ -1,11 +1,22 @@
 using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
 
 namespace FiMAdminApi.Models.Models;
 
+[Owned]
+public class StreamingConfig
+{
+    public string? Channel_Id { get; set; }
+    public string? Channel_Type { get; set; }
+}
+
 public class TruckRoute
 {
     [Key]
     public int Id { get; set; }
     
     public required string Name { get; set; }
+
+    public StreamingConfig? Streaming_Config { get; set; }
```

Comment:
```suggestion
    public StreamingConfig? StreamingConfig { get; set; }
```


Let's follow the standard .NET naming convention here, the ORM will automatically convert it to snake_case in the queries

Reply:
corrected